.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ commands:
           name: Install Dependencies
           command: |
             sudo apt-get update
-            sudo apt-get install -y cmake curl
+            sudo apt-get install -y cmake curl clang llvm
```

Comment:
Since the client needs to be in a separate workspace then you'll need to update and modify the CI config to actually run and test the client code, since it currently only tests the top-level workspace in CI IIRC.

Reply:
good call!